docs/general/events.md

```diff
@@ -43,18 +44,20 @@ that identify the class of Events but not the instance of the Event.
 <!-- semconv event -->
 | Attribute  | Type | Description  | Examples  | Requirement Level |
 |---|---|---|---|---|
-| `event.domain` | string | The domain identifies the business context for the events. [1] | `browser` | Required |
-| `event.name` | string | The name identifies the event. | `click`; `exception` | Required |
+| `event.name` | string | Consisting of a `namespace` and `name`; uniquely identifies the event. | `browser.mouse.click`; `browser.exception` | Required |
 
-**[1]:** Events across different domains may have same `event.name`, yet be unrelated events.
+**[1]:** Events across different namespaces may have same name, yet be unrelated events.
 
-`event.domain` has the following list of well-known values. If one of them applies, then the respective value MUST be used, otherwise a custom value MAY be used.
+The `namespace` portion of the name has the following list of well-known values. If one of them applies, then the respective value MUST be used, otherwise a custom value MAY be used.
 
 | Value  | Description |
 |---|---|
 | `browser` | Events from browser apps |
 | `device` | Events from mobile apps |
 | `k8s` | Events from Kubernetes |
```

Comment:
I think we should get rid of this and instead say that event namespaces are shared with all other semconv namespaces.

Reply:
I removed this table and blurb for now and I think the following section pointing to the attribute naming is enough for now. I do think we will probably have a table in here at some point for well known namespace prefixes once we come to an agreement in the WG.